<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmationBox</class>
 <widget class="QDialog" name="ConfirmationBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {color: qlineargradient(spread:pad, x1:0 y1:0, x2:1 y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));

 background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 rgb(39, 46, 141) , stop:0.5  rgb(39, 46, 141) ,  stop:1 rgb(149, 155, 173)    );
}
QPushButton {
	font: 600 20pt &quot;Times New Roman&quot;;
	padding: 5px;
	background-color: rgb(255, 255, 255);
	border: 1px solid black;
	border-radius:5px;
}
QPushButton:hover {
	font: 600 20pt &quot;Times New Roman&quot;;
	padding: 5px;
	background-color: rgb(215, 215, 215);
	border: 2px solid black;
	border-radius:5px;
}
QLabel {
	font: 600 18pt &quot;Times New Roman&quot;;
	padding: 8px; 
	color:white;
}

QLCDNumber{
	font: 600 18pt &quot;Times New Roman&quot;;
	padding: 8px; 
	background-color:transparent;
	border:0px solid transparent;
}

QPushButton:disabled {
	background-color:#ff0000;
	border:0px solid transparent;
} </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_Title">
     <property name="styleSheet">
      <string notr="true">color:white;
font:700 36pt &quot;Old English Text MT&quot;;
text-decoration:underline;</string>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Message">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font:600 18pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_No">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
